### PR TITLE
Added scaffolding

### PR DESCRIPTION
**Pull request by marianoaguero**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Lines starting with '#' are comments.
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in the repo.
+# * @lmadrazo @mdobratinich @marianoaguero
```
Comment (estebandres) [inline .github/CODEOWNERS]: ?

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# v0.0.0
```
Comment (estebandres) [inline CHANGELOG.md:1]: Does this changelog gnerate automatically? If not I would recommend using semantic commits, and automatic tag versioning with a simple gradle custom task.

File: code_of_conduct.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
+
```
Comment (estebandres) [inline code_of_conduct.md]: I would recommend to have this and the README on other commit. A simple reset and re-commit will do the trick.

**Review by estebandres — CHANGES_REQUESTED** [review]
Just a few suggestions...nothing mandatory

File: CHANGELOG.md (hunk 2 above)
Comment (marianoaguero) [reply CHANGELOG.md:1]: I added a task to do it.

**Review by estebandres — APPROVED** [review]

**Review by miglierinicarlos — APPROVED** [review]
No veo problemas en los cambios

**Pull request merged**